app/models/course.rb

```diff
@@ -371,6 +368,14 @@ def scholarship_amount
     dfe_subjects&.first&.scholarship_amount
   end
 
+  def self_accredited?
+    accrediting_provider_id.blank?
+  end
+
+  def provider_is_a_scitt?
+    provider.scitt == 'Y'
```

Comment:
```suggestion
    provider.scitt?
```

Reply:
Is it even worth having a separate method for this? I feel like `provider.scitt?` is pretty clear about it's intentions.

Reply:
Leaving as discussed.